### PR TITLE
Introduce Integration image to CBG

### PR DESCRIPTION
**Pull request by rsynek**

### CODE REVIEW

File: profiles/default/conf/realm.json [hunk 1]
```diff
@@ -1621,17 +1649,19 @@
...
     "_browser_header.xContentTypeOptions" : "nosniff",
     "bruteForceProtected" : "false",
-    "maxFailureWaitSeconds" : "900",
     "_browser_header.contentSecurityPolicy" : "frame-src 'self'",
-    "minimumQuickLoginWaitSeconds" : "60",
     "waitIncrementSeconds" : "60"
   },
-  "keycloakVersion" : "2.5.1.Final"
-}
+  "keycloakVersion" : "3.1.0.Final"
```
Comment (ibek) [inline profiles/default/conf/realm.json:1666]: When you update Keycloak version please do it so in the dockerfiles of Business Central and Process Server too.

https://github.com/edge-of-tomorrow/cloud-bpm-grid/blob/master/images/business-central/src/main/docker/Dockerfile#L13
https://github.com/edge-of-tomorrow/cloud-bpm-grid/blob/master/images/process-server/src/main/docker/Dockerfile#L23

File: bin/cbg-process-server [hunk 2]
```diff
@@ -24,6 +24,25 @@ COMMANDS:
     push \t Push Process Server Docker image to the Docker Hub
 """
 
+function waitForProcessServerToStart {
```
Comment (ibek) [inline bin/cbg-process-server]: Could you put this waiting block into cbg-integration? cbg-process-server doesn't necessary need to wait there but cbg-integration does. Imagine how would you want it behave when you only start the Process Server without Integration Routes.

Comment (rsynek) [reply bin/cbg-process-server]: Good point, will fix.

File: profiles/default/conf/realm.json (hunk 1 above)
Comment (rsynek) [reply profiles/default/conf/realm.json:1666]: Thanks for the tip.

**Comment by rsynek** [comment]
Both have been fixed, thanks for reviewing!

**Comment by ibek** [comment]
Thank you Radovan

**Pull request merged**